src/shim/telemetryClient.ts

```diff
@@ -224,12 +228,13 @@ export class TelemetryClient {
     }
 
     /**
+     * @deprecated
      * Automatically populate telemetry properties like RoleName when running in Azure
      *
      * @param value if true properties will be populated
      */
     public setAutoPopulateAzureProperties() {
-        // TODO: Config is only used during initialization of ResourceManager so it cannot be set after.
+        // NO-OP
```

Comment:
We want to support this, it should read some env variables and populate cloud role and role name, if you can create a task to follow up on this it would be great